src/arith/const_int_bound.cc

```diff
@@ -369,6 +370,8 @@ class ConstIntBoundAnalyzer::Impl
       return VisitLeftShift(op);
     } else if (op->op.same_as(tir::builtin::bitwise_and())) {
       return VisitBitwiseAnd(op);
+    } else if (op->op.same_as(tir::builtin::vscale()) && TargetHasSVE()) {
+      return MakeBound(1, 16);
```

Comment:
nit: we could make the upper bound the length of `kAArch64VScaleValues` incase more values are added in the future. Happy for this to be added in a later patch though